api/v1alpha1/catalog_types.go

```diff
@@ -16,43 +19,27 @@ const (
 
 // CatalogSpec defines the desired state of Catalog.
 type CatalogSpec struct {
-	// Source is the medium from which the PluginDefinition needs to be fetched
-	Source CatalogSource `json:"source"`
-	// Overrides are the PluginDefinition overrides to be applied
-	// +Optional
-	Overrides []CatalogOverrides `json:"overrides,omitempty"`
+	Sources []CatalogSource `json:"sources"`
 }
 
 type CatalogSource struct {
-	// Git is the Git repository source for the PluginDefinition Catalog
-	Git GitSource `json:"git"`
-	// Path is the path within the repository where the ClusterPluginDefinition / PluginDefinition Catalog is located
-	// an empty path indicates the root of the repository
-	// +Optional
-	Path string `json:"path,omitempty"`
-}
-
-type CatalogOverrides struct {
-	// Name is the name of the PluginDefinition to patch with an alias
-	Name string `json:"name"`
-	// Alias is the alias to apply to the PluginDefinition Name via Kustomize patches
-	// For SourceType Helm, this field is passed to postRender Kustomize patch
-	Alias string `json:"alias"`
-	// Repository is the repository to override in the PluginDefinition .spec.helmChart.repository
-	// +Optional
-	Repository string `json:"repository,omitempty"`
+	// Repository - the Git repository URL
+	Repository string `json:"repository"`
 
-	// TODO: implement Options in Overrides for further values patching in PluginDefinition || ClusterPluginDefinition
-}
+	// Resources - list of path to PluginDefinition file
```

Comment:
CRD doc strings are so far all full sentences. 
```suggestion
	// Resources contains the list of path to PluginDefinition files
```